.github/workflows/release-docker-images.yml

```
@@ -10,7 +10,7 @@ on:
     inputs:
       release_version:
         description: 'Comma-separated list of versions to build, e.g. v6.0,v6.1'
```

```suggestion
        description: 'Comma-separated list of versions to build, e.g. v6.0,v6.1,v6.2'
```
Could also include the new version in the example list.